.circleci/config.yml

```diff
@@ -1037,60 +893,43 @@ workflows:
               only:
                 - /^test-[\d|.]+$/
                 - /^release-[\d|.]+$/
-      #Todo: solid publishing mechanism.
-      #- docker-ci-publish:
-      #    context: docker
-      #    requires:
-      #      - dev-setup-changes
-      #    filters:
-      #      branches:
-      #        only:
-      #          - master
-      - dev-setup-changes:
+      - docker-ci-build:
+          context: docker
+          base-image: circleci
           filters:
             branches:
-              ignore:
-                - auto
-                - /^test-[\d|.]+$/
-                - /^release-[\d|.]+$/
-      # These builds test the dev_setup.sh
+              only:
+                - master
       - docker-ci-build:
-          requires:
-            - dev-setup-changes
+          base-image: circleci
           filters:
             branches:
               ignore:
-                - auto
                 - /^test-[\d|.]+$/
                 - /^release-[\d|.]+$/
                 - master
-      - dev_setup-test-centos:
-          requires:
-            - dev-setup-changes
+      - docker-ci-build:
+          base-image: centos
```

Comment:
Why are we building these variants?

Reply:
to test the dev_setup.sh that claims to support centos.  now we know.